app/controllers/ops_controller.rb

```diff
@@ -741,21 +741,21 @@ def rebuild_toolbars(presenter)
     presenter[:record_id] = determine_record_id_for_presenter
   end
 
-  def handle_bottom_cell(nodetype, presenter, r, locals)
+  def handle_bottom_cell(presenter, r, locals)
     # Handle bottom cell
-    if nodetype == "log_depot_edit"
-      presenter.hide(:form_buttons_div, :pc_div_1)
-    elsif @pages || @in_a_form
+    if @pages || @in_a_form
       if @pages
-        presenter.hide(:form_buttons_div).show(:pc_div_1)
```

Comment:
The changes from here on (except for that `x_edit_buttons` -> `angular/paging_div_buttons`) look like a bad rebase, we do want `presenter.hide` & `show` & `update` instead of `[:set_visible_elements]` and `[:update_partials]`.